### DISCUSSION TITLE
Will there be a significant difference in the performance of randomly writing to the test disk when setting or not setting the ODIRECT field when opening a file?

### DISCUSSION THREAD

**Discussion by wwwwxxxxhhhh** [axboe/liburing#1152] [Q&A]
When I didn't set up ODIRECT, the performance tested was 300W iops. Under the same traffic model, the performance tested with ODIRECT was 30W iops. Using the fio tool, the performance tested with ODIRECT was around 60W iops. This may be an error in my testing program? Or what is the reason?

**Comment by axboe** [comment]
This isn't a generic help forum, it's only for io_uring.

**Reply by wwwwxxxxhhhh** [reply]
I am testing io_uring and found that the performance is 30w iops when setting ODIRECT when opening files, but it can reach 300w iops when not setting ODIRECT
